numba/types.py

```diff
@@ -822,6 +822,17 @@ def key(self):
         return self.dtype, self.ndim, self.layout, self.mutable
 
 
+class ArrayCTypes(Type):
+    def __init__(self, arytype):
+        self.ndim = arytype.ndim
```

Comment:
Though, the two attributes are not implemented yet.  I think the type should still reflect that.